.github/workflows/pytest-core-mpi.yml

```diff
@@ -14,6 +14,9 @@ jobs:
   build:
     name: pytest-mpi
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ['3.9', '3.8']
```

Comment:
this is doubling the cost of `pytest-core-mpi`, which is already quite significant.

I wonder whether we should forget about 3.8 for MPI testing @mloubout @rhodrin . Other options welcome

Reply:
This will spawn a job for each concurrently, hence unless we're hitting the limit of runners we can use at any one time (which I don't think we currently are?) won't affect overall test times.

Reply:
ah, right. How can we check?